### DISCUSSION TITLE
What is the purpose of the „Inbox“ Page and how to write comments?

### DISCUSSION THREAD

**Discussion by LimesRunner** [usememos/memos#2892] [Q&A] [answered]
I just started with Memos, and I didn‘t found anything in the docs. It seems like the Input page has been introduced with v0.17 and it shall display comments from other members.

But how to write these comments? Currently the only way to see memos from other users is the (unfortunately) very restricted Explore page. From this page there seems to be no possibility to react on a memo.

**Comment by zezretro** [comment]
Bump. interested in what the inbox page does

**Comment by zezretro** [comment, marked as answer]
Looking at the code, I would guess Inbox is for notifications like when the server is upgraded and when someone else comments on your notes
